### DISCUSSION TITLE
`create` is deprecated, what is the alternative use?

### DISCUSSION THREAD

**Discussion by zhangfisher** [pmndrs/zustand#1973] [Q&A]
`create` is deprecated, what is the alternative use in v4.4.0?

Currently, the document has not been updated

**Comment by dai-shi** [comment]
Can you elaborate please? What's the exact deprecation message?

**Comment by zhangfisher** [comment]
[here](https://github.com/pmndrs/zustand/pull/1403)

`create` be deprecated on 5.0?

**Reply by dai-shi** [reply]
```js
import create from 'zustand'
```
that ☝️ is deprecated already on v4 and will be removed on v5.

you want to do this 👇 instead.
```js
import { create } from 'zustand'
```

---

I'm not sure which deprecation you face in #1403.

**Reply by itse4elhaam** [reply]
Hey, thanks for the quick answer!

**Reply by renegade93** [reply]
Thanks for this, I also had the same issue.
